excel_import_export/security/ir.model.access.csv

```diff
@@ -0,0 +1,7 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+xlsx_template_user,xlsx_template_user,model_xlsx_template,,1,1,1,1
+xlsx_template_export_user,xlsx_template_export_user,model_xlsx_template_export,,1,1,1,1
+xlsx_template_import_user,xlsx_template_import_user,model_xlsx_template_import,,1,1,1,1
+access_export_xlsx_wizard,access_export_xlsx_wizard,model_export_xlsx_wizard,base.group_user,1,1,1,1
+access_import_xlsx_wizard,access_import_xlsx_wizard,model_import_xlsx_wizard,base.group_user,1,1,1,1
+access_report_xlsx_wizard,access_report_xlsx_wizard,model_report_xlsx_wizard,base.group_user,1,1,1,1
```

Comment:
The current definition of the security rules is deprecated. See the warnings below.
```
odoo.addons.base.models.ir_model: Rule xlsx_template_user has no group, this is a deprecated feature. Every access-granting rule should specify a group. 
odoo.addons.base.models.ir_model: Rule xlsx_template_export_user has no group, this is a deprecated feature. Every access-granting rule should specify a group. 
odoo.addons.base.models.ir_model: Rule xlsx_template_import_user has no group, this is a deprecated feature. Every access-granting rule should specify a group. 
```